Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3 AS builder
+WORKDIR /code
+COPY . /code/
+RUN apk --no-cache add nodejs ruby-dev build-base git yarn \
+  &&  gem install --no-document middleman
+RUN yarn install --immutable \
+  && yarn run lint \
```

Comment:
is this just for build, you don't need `lint` and `test`

Reply:
Removed build part from container in 9c8f130